app/services/forest_liana/search_query_builder.rb

```diff
@@ -50,7 +50,7 @@ def acts_as_taggable_query(tagged_records)
     end
 
     def search_param
-      if @search
+      if (@search && !@search.empty?())
```

Comment:
Can you explain the fix for the review in Trello or Github?
It is quite hard to follow you reflexion based on this line of code.

Reply:
The issue is that when you want your collections, rails crash with an error `unknown function to_i` because search is actually empty (search = {}). So I just check if search is empty first before computing any search params

Reply:
Actually the issue is more generic than only the `search`. Any collection name that could interfere with query params can raise issues:

<img width="1595" alt="screen shot 2018-08-21 at 12 06 33" src="https://user-images.githubusercontent.com/664261/44395689-edbbe380-a53a-11e8-857e-c0f1c45060f5.png">

So `pages`, `searches`, `fields`, `sorts`, `timezones` are table name that could generate the same kind of error.
We need to find a way to prevent Rails (I feel that this is Rails that does that) to add `"collectionName" => {}` in the params.